### PR TITLE
feat: add language parameter

### PR DESCRIPTION
**Pull request by anistouri**

### CODE REVIEW

File: src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js [hunk 1]
```diff
@@ -36,10 +37,10 @@ const useDisplayView = (network, studyUuid, currentNode) => {
     const substationLayout = useSelector(
         (state) => state[PARAM_SUBSTATION_LAYOUT]
     );
-
```
Comment (bendaamerahm) [inline src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js:39]: remove this change

File: src/components/dialogs/connectivity-edition.js [hunk 2]
```diff
@@ -122,6 +123,7 @@ export const useConnectivityValue = ({
     const [isDiagramPaneOpen, setIsDiagramPaneOpen] = useState(false);
     const intl = useIntl();
     const currentNode = useSelector((state) => state.currentTreeNode);
+    const language = useSelector((state) => state[PARAM_LANGUAGE]);
```
Comment (bendaamerahm) [inline src/components/dialogs/connectivity-edition.js:126]: i think this need to be declared in position-diagram-pane not here

File: src/components/dialogs/connectivity-edition.js [hunk 3]
```diff
@@ -325,6 +327,7 @@ export const useConnectivityValue = ({
                     onClose={handleCloseDiagramPane}
                     voltageLevelId={voltageLevelObjOrId}
                     currentNodeUuid={currentNodeUuid}
+                    language={language}
```
Comment (bendaamerahm) [inline src/components/dialogs/connectivity-edition.js:330]: same in position-diagram-pane not here

File: src/components/dialogs/connectivity-edition.js [hunk 4]
```diff
@@ -344,6 +347,7 @@ export const useConnectivityValue = ({
         isDiagramPaneOpen,
         voltageLevelObjOrId,
         currentNodeUuid,
+        language,
```
Comment (bendaamerahm) [inline src/components/dialogs/connectivity-edition.js:350]: same in position-diagram-pane not here

**Review by bendaamerahm — CHANGES_REQUESTED** [review]

File: src/utils/rest-api.js [hunk 5]
```diff
@@ -317,6 +320,7 @@ export function getSubstationSingleLineDiagram(
             ...(componentLibrary !== null && {
                 componentLibrary: componentLibrary,
             }),
+            language,
```
Comment (EstherDarkish) [inline src/utils/rest-api.js:323]: Here why don't we define this param as `language: language,` instead ?

**Review by FranckLecuyer — APPROVED** [review]

**Review by bendaamerahm — APPROVED** [review]

**Pull request merged**